src/marlin_stubs/sdcard/M20-M30_M32-M34.cpp

```diff
@@ -119,7 +121,12 @@ void GcodeSuite::M29() {
 
 // M30 - Delete a file on the SD card
 void GcodeSuite::M30() {
-    // TODO
+    ArrayStringBuilder<FILE_PATH_MAX_LEN> filepath;
```

Comment:
FILE_PATH_MAX_LEN really only considers SFN paths, so it will quite possibly not be able to fit LFNs.

Reply:
ok, using FF_MAX_LFN instead.